common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -818,6 +822,8 @@ def _get_answer_notification(self, render_notifications):
                     ).format(progress=str(progress))
                 else:
                     answer_notification_message = _('Partially Correct')
+            elif answer_notification_type == 'answered':
+                answer_notification_message = self.submitted_message
```

Comment:
Since the [previously unsupported implementation uses `submitted_message`](https://openedx.atlassian.net/browse/DOC-1319?focusedCommentId=193723&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-193723), I'm inclined to change all of the `answered` types to `submitted`.

This means changing https://github.com/open-craft/edx-platform/pull/58's code too, but it's worth it while we're in the early stages.